CHANGELOG.md

```diff
@@ -1,13 +1,19 @@
 ## Unreleased
+- 
 
+## 7.1.0
 ### Improvements
 
 - Fix appearance of borders in segmented "Big" button groups. ([#359](https://github.com/18F/identity-design-system/pull/359))
 - Improve compilation time for default package by upwards of 50%. ([#356](https://github.com/18F/identity-design-system/pull/356))
+- Upgrade Ruby and Puppeteer ([#367](https://github.com/18F/identity-design-system/pull/367) and [#364](https://github.com/18F/identity-design-system/pull/364))
+- LG-10522: Add Icon List Component ([#368](https://github.com/18F/identity-design-system/pull/368))
+- LG-10523: Add Image Example Component ([#369](https://github.com/18F/identity-design-system/pull/369))
```

Comment:
I think we should avoid references to the ticket, and move under a "New Features" section to follow precedent from earlier releases.

```suggestion
- Add custom styling for the Icon List component. ([#368](https://github.com/18F/identity-design-system/pull/368))
- Add new Image Example component. ([#369](https://github.com/18F/identity-design-system/pull/369))
```

Reply:
Addressed in f77d2ccf473d2cde9ce1364888a44b98180145cb.